cpp/src/interop/from_arrow_device.cu

```diff
@@ -158,12 +155,15 @@ dispatch_tuple_t dispatch_from_arrow_device::operator()<cudf::string_view>(
       {});
   }
 
-  auto offsets_view = column_view{data_type(type_id::INT32),
+  data_type offsets_type(type_id::INT32);
+  if (schema->type == NANOARROW_TYPE_LARGE_STRING) { offsets_type = data_type(type_id::INT64); }
+  auto offsets_view = column_view{offsets_type,
```

Comment:
@davidwendt since we're making a view and not a column here, is this approach to supporting large strings sufficient? I think it might be, since it's only in the owning case where we have to deal with forcing usage of a buffer manually instead of using normal column constructors, but want to be sure we're not overlooking anything.

Reply:
This looks right to me. This is consistent with my understanding of Arrow large strings.